packages/idyll-components/src/desmos.js

```diff
@@ -95,8 +95,8 @@ class Desmos extends React.Component {
     });
   }
 
-  componentWillUpdate(nextProps) {
-    const { equation } = nextProps;
+  componentDidUpdate(prevProps) {
+    const { equation } = prevProps;
     // Only instantiate & update the calculator
     // when necessary to improve performance.
     if (
```

Comment:
Oops - just found a small issue with this:

when it ends up rendering the equation on line `112`, it uses the equation from `prevProps`, but we really want it to use the latest `equation` available, otherwise the rendering is always a tick behind